internetarchive/openlibrary
# Fix redundant DB table creation attempts in local environment

Closes #5517


Removes scripts that attempt to create tables that were already created in local `db_1` instances.

### Technical

All redundant table and constraint creation scripts have been removed from `scripts/dev-instance/dev_db.pg_dump`.  Redundant inserts are also removed from that file.

All queries that alter ownership of tables and sequences have not been removed.

### Testing

1. If Docker is built, remove DB volume.  Otherwise, build Docker.
2. Run `docker-compose up` and look for any errors in `db_1`
3. Ensure that reading log functionality and ratings are working as expected.

### Screenshot


### Stakeholders
